src/backend/distributed/planner/multi_explain.c

```diff
@@ -108,6 +109,11 @@ MultiExplainOneQuery(Query *query, IntoClause *into, ExplainState *es,
 	instr_time planDuration;
 	Query *originalQuery = NULL;
 	RelationRestrictionContext *restrictionContext = NULL;
+#if PG_VERSION_NUM >= 90600
+	int cursorOptions = into ? 0 : CURSOR_OPT_PARALLEL_OK;
+#else
+	int cursorOptions = 0;
```

Comment:
Hm, very slight preference of moving the ifdef block away from the variable declaration. E.g. rather do something like
```
int cursorOptions = 0;

#if PG_VERSION_NUM >= 90600
/*
 * Allow parallel plans in 9.6+ unless selecting into a table. That's important because we're otherwise breaking 
 * explain for ! citus plans.
 */ 
if (!into)
{
    cursorOptions |= CURSOR_OPT_PARALLEL_OK
}
#endif
```